lib/cli/commands/listswaps.ts

```diff
@@ -0,0 +1,27 @@
+import { callback, loadXudClient } from '../command';
+import { Arguments } from 'yargs';
+import { ListSwapsRequest } from '../../proto/xudrpc_pb';
+
+export const command = 'listswaps [limit] [failed]';
```

Comment:
`failed` does not match up with `status`. I'm thinking `status` should be renamed anyhow (see my comment below), but however they end up this line ought to have accurate naming.